.github/scripts/cmd/cmd.py

```diff
@@ -0,0 +1,188 @@
+}
+
+parser = argparse.ArgumentParser(prog="/cmd ", description='A command runner for polkadot runtimes repo', add_help=False)
+parser.add_argument('--help', action=_HelpAction, help='help for help if you need some help')  # help for help
+
+subparsers = parser.add_subparsers(help='a command to run', dest='command')
+
+"""
+BENCH 
+"""
+
+bench_example = '''**Examples**:
+
+ > runs all benchmarks
+ 
+ %(prog)s
+ 
+ > runs benchmarks for pallet_balances and pallet_multisig for all runtimes which have these pallets
+ > --quiet makes it to output nothing to PR but reactions
+ 
+ %(prog)s --pallet pallet_balances pallet_xcm_benchmarks::generic --quiet
+ 
+ > runs bench for all pallets for polkadot runtime and continues even if some benchmarks fail
+ 
+ %(prog)s --runtime polkadot --continue-on-fail 
+ 
+ > does not output anything and cleans up the previous bot's & author command triggering comments in PR 
+ 
+ %(prog)s --runtime polkadot kusama --pallet pallet_balances pallet_multisig --quiet --clean 
+
+ '''
+
+parser_bench = subparsers.add_parser('bench', help='Runs benchmarks', epilog=bench_example, formatter_class=argparse.RawDescriptionHelpFormatter)
+
+for arg, config in common_args.items():
+    parser_bench.add_argument(arg, **config)
+
+parser_bench.add_argument('--runtime', help='Runtime(s) space separated', choices=runtimeNames, nargs='*', default=runtimeNames)
+parser_bench.add_argument('--pallet', help='Pallet(s) space separated', nargs='*', default=[])
+
+"""
+FMT 
+"""
+parser_fmt = subparsers.add_parser('fmt', help='Formats code')
+for arg, config in common_args.items():
+    parser_fmt.add_argument(arg, **config)
+
+args, unknown = parser.parse_known_args()
+
+print(f'args: {args}')
+
+if args.command == 'bench':
+    tempdir = tempfile.TemporaryDirectory()
+    print(f'Created temp dir: {tempdir.name}')
+    runtime_pallets_map = {}
+    failed_benchmarks = {}
+    successful_benchmarks = {}
+
+    profile = "release"
+
+    print(f'Provided runtimes: {args.runtime}')
+    # convert to mapped dict
+    runtimesMatrix = list(filter(lambda x: x['name'] in args.runtime, runtimesMatrix))
+    runtimesMatrix = {x['name']: x for x in runtimesMatrix}
+    print(f'Filtered out runtimes: {runtimesMatrix}')
+
+    # loop over remaining runtimes to collect available pallets
+    for runtime in runtimesMatrix.values():
+        os.system(f"cargo build -p {runtime['package']} --profile {profile} --features runtime-benchmarks")
+        print(f'-- listing pallets for benchmark for {runtime["name"]}')
+        wasm_file = f"target/{profile}/wbuild/{runtime['package']}/{runtime['package'].replace('-', '_')}.wasm"
+        output = os.popen(
+            f"frame-omni-bencher v1 benchmark pallet --no-csv-header --all --list --runtime={wasm_file}").read()
+        raw_pallets = output.split('\n')
+
+        all_pallets = set()
+        for pallet in raw_pallets:
+            if pallet:
+                all_pallets.add(pallet.split(',')[0])
```

Comment:
```suggestion
                all_pallets.add(pallet.split(',')[0].trim())
```
Just in case, i remember there was one version that would add spaces.